lib/node_modules/@stdlib/math/base/ops/cneg/docs/types/index.d.ts

```diff
@@ -20,49 +20,57 @@
 
 /// <reference types="@stdlib/types"/>
 
-import { ArrayLike } from '@stdlib/types/array';
+import { Complex128 } from '@stdlib/types/object';
 
 /**
-* Negates a complex number.
+* Negates a double-precision complex floating-point numbers.
 *
-* @param out - output array
-* @param re - real component
-* @param im - imaginary component
-* @returns real and imaginary components
+* @param z - complex number
+* @returns result
 *
 * @example
-* var Float32Array = require( `@stdlib/array/float32` );
+* var Complex128 = require( '@stdlib/complex/float64' );
+* var real = require( '@stdlib/complex/real' );
+* var imag = require( '@stdlib/complex/imag' );
+* var cneg = require( '@stdlib/math/base/ops/cneg' );
```

Comment:
```suggestion
* var Complex128 = require( `@stdlib/complex/float64` );
* var real = require( `@stdlib/complex/real` );
* var imag = require( `@stdlib/complex/imag` );
```

Because of a quirk in our lint tooling, we need to use template literals for `require` paths in TypeScript declaration examples.